### ISSUE TITLE
Error with tool tip builder with latest update. 

### ISSUE THREAD

**Issue by SamerDiab** [appreciated/apexcharts-flow#49]
```Uncaught TypeError: Cannot read property 'formatter' of undefined
    at HTMLElement.updateConfig (apexcharts-wrapper.js?3539:196)
    at HTMLElement.ready (apexcharts-wrapper.js?3539:104)
    at HTMLElement.Ww.s.ready (client-4FB121A4CC0177A1068809F06428A755.cache.js:999)
    at HTMLElement._enableProperties (properties-changed.js?2b39:333)
    at HTMLElement.connectedCallback (properties-mixin.js?2f2d:214)
    at HTMLElement.connectedCallback (element-mixin.js?301f:619)
    at Xv (client-4FB121A4CC0177A1068809F06428A755.cache.js:955)
    at nw (client-4FB121A4CC0177A1068809F06428A755.cache.js:918)
    at sy.ty [as _] (client-4FB121A4CC0177A1068809F06428A755.cache.js:999)
    at YA (client-4FB121A4CC0177A1068809F06428A755.cache.js:914)
    at xr (client-4FB121A4CC0177A1068809F06428A755.cache.js:994)
    at Nr.Or [as D] (client-4FB121A4CC0177A1068809F06428A755.cache.js:999)
    at ek (client-4FB121A4CC0177A1068809F06428A755.cache.js:407)
    at pr (client-4FB121A4CC0177A1068809F06428A755.cache.js:995)
    at qr (client-4FB121A4CC0177A1068809F06428A755.cache.js:979)
    at Dt.Ft [as vb] (client-4FB121A4CC0177A1068809F06428A755.cache.js:999)
    at BB.CB [as K] (client-4FB121A4CC0177A1068809F06428A755.cache.js:999)
    at XMLHttpRequest.<anonymous> (client-4FB121A4CC0177A1068809F06428A755.cache.js:585)
    at sb (client-4FB121A4CC0177A1068809F06428A755.cache.js:434)
    at vb (client-4FB121A4CC0177A1068809F06428A755.cache.js:865)
    at XMLHttpRequest.<anonymous> (client-4FB121A4CC0177A1068809F06428A755.cache.js:599)```


This is the javascript error.

**Comment by appreciated** [comment]
Please add some code to reproduce this issue.

**Comment by appreciated** [comment]
Fixed by e0f68054f61f26fab0c3999f9c77f494bf88d8e1
